SOC-1820

jsutterfield opened the pull request:
There were 2 problems with Special:Coppa, which we use to blank out underage user's information.

The first was fixed in SOC-1820, which this is branched off of. There was a bug in the logic we used to save `null` attributes. By fixing SOC-1820, Special:Coppa now deletes all the user's attributes from the attribute service as expected.

The second was an issue with `UserIdentityBox::resetUserProfile` which is what this branch specifically addresses. "Blank user profiles" on Special:Coppa was timing out because 2 database transactions were trying to operate on the same table. One transaction updates the timestamp for a row, and the second transaction deletes the row. The transaction to update the timestamp (called from `$user->saveSettings();`) doesn't do an explicit commit when it's done, so when the second transaction to delete the row comes along, the table is locked creating a deadlock.

The second transaction however (the one to delete the row), commits as soon as it's done. The fix then to swap the order of these transactions so that the one that actually commits runs first. The order doesn't matter b/c the end result is the same, the row is deleted.

Ticket: https://wikia-inc.atlassian.net/browse/SOC-1840

drets committed:
ADEN-2712 Refactor floating medrec

drets committed:
ADEN-2712 CR fixes

drets committed:
ADEN-2712 CR fixes 2

drets committed:
ADEN-2712 CR fixes 3

drets committed:
ADEN-2712 Introduce ads-on-event-friendly FMR

drets committed:
ADEN-2712 Move call of FMR to Rail

drets committed:
ADEN-2712 Enable FMR after crossing start position

Grunny committed:
(SEC-49) Only return results for existing wikias

owend committed:
Merge pull request #9550 from Wikia/PLATFORM-1355

PLATFORM-1355: Create a special kind of logger that will help me track when the output of article becomes empty

fraszczakszymon committed:
ADEN-2820 Add new AdEngineRunner module

fraszczakszymon committed:
ADEN-2820 Create new instant global and use it in AdContext

fraszczakszymon committed:
ADEN-2820 Add new dependency

fraszczakszymon committed:
ADEN-2820 Use AdEngineRunner in desktop run

idradm committed:
DAT-3639 legacy format removed

Commit:
Merge branch 'dev' into image-review-mercury-iframe

fraszczakszymon committed:
ADEN-2820 Revert comment and fix Rubicon Fastlane context on mercury

Grunny committed:
Merge pull request #9504 from Wikia/CE-3206-i18n

CE-3206 i18n for GlobalShortcuts

idradm committed:
DAT-3639 use new format in validator and helper

idradm committed:
DAT-3639 return community data only if not empty

RafalWilinski committed:
Merge pull request #9498 from Wikia/image-review-mercury-iframe

Image-review extension embedded from Mercury

fraszczakszymon committed:
ADEN-2820 Add JSDocs

idradm committed:
DAT-3639 fix data saving

azarek committed:
Merge pull request #9312 from Wikia/CE-globalshortcuts

GlobalShortcuts hackathon idea

idradm committed:
DAT-3639 adapt tests to new format

idradm committed:
Merge branch 'dev' into DAT-3639b

# Conflicts:
#	extensions/wikia/MercuryApi/MercuryApiController.class.php

idradm committed:
DAT-3639 merge with dev

vforge committed:
XW-918 | Make sure you always get articleType

adamkarminski committed:
GlobalShortcuts Add Russian translations

Grunny committed:
Merge pull request #9555 from Wikia/CE-globalshortcuts-i18n

GlobalShortcuts Add Russian translations

Grunny committed:
(SEC-48) Add a version of serialize that disallows objects

idradm committed:
DAT-3639 curated content data methods names changes

Commit:
MAIN-6280: Add HTTP caching to Recirculation controller index

Commit:
MAIN-6280: Switch to using a standard Wikia Response cache time

Commit:
Merge pull request #9556 from Wikia/MAIN-6280

MAIN-6280: Add HTTP caching to Recirculation controller index

Commit:
SEO-246 Don't publish any more language links to lilly

Processing the queue is disabled because of an issue found, so right now
lilly serves as a read-only database of starwars links only.

This means we don't need to hook to the links processing and post
anymore interlanguage links to lilly.

Commit:
SEO-246 LillyValidator class is not needed either

Commit:
RECIRC-35: Track recirculation impressions in GA

Commit:
Merge pull request #9558 from Wikia/RECIRC-35

RECIRC-35: Track recirculation impressions in GA

Commit:
Merge pull request #9557 from Wikia/SEO-246-disable-lilly-queue-links

SEO-246 Don't publish any more language links to lilly

Commit:
PLATFORM-1355: Add little bit more logging for debugging

inez committed:
Merge pull request #9559 from Wikia/PLATFORM-1355

PLATFORM-1355: Add little bit more logging for debugging

Commit:
SOC-1840 move saveOptions out of loop

Commit:
SOC-1840 call Wikia::invalidateUser before user->saveSettings to prevent deadlocks

idradm committed:
DAT-3639 use only needed curated content data

fraszczakszymon committed:
ADEN-2820 CR fixes

idradm committed:
DAT-3639 input format changed

idradm committed:
DAT-3639 description changed

idradm committed:
DAT-3639 fix if else formatting

lukaszkonieczny committed:
CE-3273: Fix typing letter into search box

rybmat committed:
fixing regexp

azarek committed:
Merge pull request #9457 from Wikia/CE-2533

CE-2533 Enable easy review escalation

lukaszkonieczny committed:
CE-3280: Fix translations for header, footer and shortcuts

rybmat committed:
comment

rybmat committed:
ResultHelper

fraszczakszymon committed:
ADEN-2820 Remove redundant adContext.getContext()

rybmat committed:
template changes

rybmat committed:
another testcase

azarek committed:
Merge pull request #9561 from Wikia/CE-3273

CE-3273: Fix typing letter into search box

lukaszkonieczny committed:
CE-3279: use default text color

rybmat committed:
moving classes to template

rybmat committed:
inline tracking data

idradm committed:
DAT-3639 send community data feature variable to mercury

rybmat committed:
thumb tracking

Grunny committed:
Merge branch 'SEC-48-SEC-49' into dev

Commit:
Merge pull request #9565 from Wikia/DAT-3641b

Dat 3641b

Grunny committed:
Increase MAX_SERIALIZED_ARRAY_DEPTH to 8

lukaszkonieczny committed:
CE-3279: Add colors for input field

Grunny committed:
Merge branch 'serialize-depth' into dev

drets committed:
ADEN-2712 Support IE11

azarek committed:
Merge pull request #9564 from Wikia/CE-3280

CE-3280: Fix translations for header, footer and shortcuts

lukaszkonieczny committed:
CE-3283: Close modals after shortcut is executed

lukaszkonieczny committed:
CE-3283: Check if modals exists

azarek committed:
Merge pull request #9566 from Wikia/CE-3279

CE-3279: Use proper colors in global shortcuts modal

lukaszkonieczny committed:
CE-3285: Remove double escaping

bognix committed:
XW | remove link to WAM on pt wiki - it does not exist

vforge committed:
Merge pull request #9569 from Wikia/XW-messages-fix

XW | remove link to WAM on pt wiki - it does not exist

Commit:
Merge pull request #9553 from Wikia/ADEN-2820-delay-adengine

ADEN-2820 Delay AdEngine

bognix committed:
XW-1009 show smartbanner for app on ios

Commit:
Merge pull request #9530 from Wikia/SOC-1820

SOC-1820 Delete attributes from service with value set to null.

idradm committed:
DAT-3639 fix community data getter

bognix committed:
XW-1009 fallback to My Wikia App if there is no dedicated app

Commit:
SOC-1840 remove avatar from avatar service as well

jsutterfield commented on extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php:
```diff
@@ -28,6 +28,7 @@ class UserIdentityBox {
 
 	public $optionsArray = array(
 		'location',
+		'bio',
```
`bio` is used by apps for their profiles. By adding this here, it's included in the attributes to be blanked out by `resetUserProfile`.

Commit:
PLATFORM-1355: Temporary disable some logging and create a new logging to track down if we are passing correct data to Mustache template

owend committed:
Merge pull request #9572 from Wikia/PLATFORM-1355

PLATFORM-1355: Temporary disable some logging and create a new logging to track down if we are passing correct data to Mustache template

armonr commented on extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php:
```diff
@@ -577,11 +578,16 @@ public function resetUserProfile() {
 			}
 			$this->user->setGlobalAttribute( $option, null );
-
-			$this->user->saveSettings();
```
It's curious why this was being called in the loop!

armonr commented:
:shipit:

Commit:
SEO-252 Interwiki links sometimes add an extra "/Wiki/"

The problematic part is the %2F in the URL. Apache treats that as an
incorrect URL
(http://httpd.apache.org/docs/2.2/mod/core.html#allowencodedslashes) and
fires the 404 handler. Before the changes the ErrorDocument was
index.php?title=Special:Our404Handler and the title was ignored if the
(URL-decoded) QUERY_STRING matched the MediaWiki $wgArticlePath.

With the updated code the 404 goes to the /redirect-canonical.php which
takes you to /wiki/(the-original-path). This code was not meant to
handle requests with /wiki/ in PATH.

We're fixing the redirect done in SpecialInterwikiDispatcher_body.php by
replacing the PHP's urlencode with MediaWiki's wfUrlencode, which
encodes fewer characters (for instance it leaves / and : intact).

Also we can extend the redirect-canonical.php to also work in the case
it's called as a broken-URL handler.

A comment mentioning the %2F breaks URLs was updated to point to the
Apache docs on AllowEncodedSlashes.

Redirect-canonical.php was also updated to output a X-Redirected-By
header (with a special version for %2F version) for easier debugging of
similar issues in the future.

lukaszkonieczny committed:
CE-3247: Check if userpage exists

lukaszkonieczny committed:
CE-3247: Better check if userpage is Title object

Commit:
Merge pull request #9563 from Wikia/DAT-3623

Dat 3623

idradm committed:
DAT-3639 temporary fix for mercury side validation

idradm committed:
DAT-3639 use new format in validator controller

azarek committed:
Merge pull request #9567 from Wikia/CE-3283

CE-3283: Close modals after shortcut is executed

fraszczakszymon committed:
ADEN-2292 OpenX provider removal

vforge committed:
Merge pull request #9571 from Wikia/XW-1009

XW-1009 show smartbanner for app on ios

idradm committed:
DAT-3639 use label as main, fallback to title if doesn't exists

Commit:
Merge pull request #9433 from Wikia/ADEN-2712-floating-medrec

ADEN-2712 Refactor floating medrec

idradm committed:
DAT-3639 umigrate image crop as well

macbre committed:
PhalanxService: log POST data when service does not respond correctly

Commit:
Merge pull request #9576 from Wikia/SUS-17-log-phalanx-post-data

PhalanxService: log POST data when service does not respond correctly

Commit:
SUS-58 Add backlinks as PhalanxStats subtitle

lukaszkonieczny committed:
CE-3289: Remove shortcuts entry point from my tools

lukaszkonieczny committed:
CE-3289: Select first suggestion

Commit:
Merge pull request #9577 from Wikia/SUS-58

SUS-58 Add backlinks as PhalanxStats subtitle

idradm committed:
DAT-3639 look up community data only if correct wiki id provided

Commit:
SUS-68 Do not call getUniqueTitle() statically

lukaszkonieczny committed:
CE-3288: Remove ImageReviewMercury from special pages list

lukaszkonieczny committed:
CE-3288: Add short description for Special:Templates

lukaszkonieczny committed:
CE-3288: Make suggestions case insensitive

Commit:
SUS-85 Set the format of a response upfront

sebastianmarzjan committed:
Merge pull request #9520 from Wikia/DAT-3639b

Dat 3639b use curated content data service

vforge committed:
Merge remote-tracking branch 'origin/dev' into XW-918

Commit:
fix broken link on Special:PhalanxStats by changing the MW message to match custom message overwrite on community.wikia.com

vforge committed:
Merge pull request #9554 from Wikia/XW-918

XW-918 | Make sure you always get articleType

lukaszkonieczny committed:
CE-3288: Add italian translations

lukaszkonieczny committed:
CE-3288: Change wiki to wikia

lukaszkonieczny committed:
Merge pull request #9575 from Wikia/CE-3247

CE-3247: Check if userpage exists

Commit:
PLATFORM-1828: Log each and every queued task

Grunny committed:
(CE-775) Exclude non-existent users on Special:ListUsers

Make sure we don't list non-existent users on Special:ListUsers. Prevent
rows with empty usernames in the events table from being returned which
will catch the majority of these issues, and fix the check for whether or
not the user exists so it works properly.

Commit:
SUS-85 Fix Lightbox unit tests

Commit:
Add queue name to logs

Commit:
Merge pull request #9583 from Wikia/task-queue-logging

PLATFORM-1828: Log each and every queued task

Commit:
SUS-48, remove LVS

adamkarminski committed:
Merge pull request #9581 from Wikia/CE-3288

CE-3288: Fix translations in Global Shortcuts

Grunny committed:
Merge pull request #9584 from Wikia/CE-775

(CE-775) Exclude non-existent users on Special:ListUsers

lukaszkonieczny committed:
CE-3290: Fix shortcuts on French keyboard

adamkarminski committed:
Merge pull request #9587 from Wikia/CE-3290

CE-3290: Fix shortcuts on French keyboard

Commit:
Make log params not collide with task runner

adamkarminski committed:
CE-3289 Remove an unneeded comma

owend committed:
Merge pull request #9574 from Wikia/SEO-252-interwiki-links-add-wiki

SEO-252 Interwiki links sometimes add an extra "/Wiki/"

garthwebb commented:
LGTM

Commit:
Merge pull request #9580 from Wikia/SUS-85

SUS-85 Set the format of a response to JSON upfront

lukaszkonieczny committed:
CE-3292: Added translations for some special pages

drets committed:
Merge pull request #9545 from Wikia/ADEN-2873-ox-everywhere

ADEN-2873 Remove lifestyle restriction

azarek committed:
Merge pull request #9568 from Wikia/CE-3285

CE-3285: Remove double escaping

Commit:
SUS-75 Check type of a file supplied to the Video Embed Tool

azarek committed:
Merge pull request #9578 from Wikia/CE-3289

CE-3289: Remove shortcuts entry point from my tools

Commit:
Merge pull request #9579 from Wikia/SUS-68

SUS-68 Do not call getUniqueTitle() statically

lukaszkonieczny committed:
CE-2505: Do not allow move js pages to MW or USER namespace

Grunny committed:
Merge pull request #9592 from Wikia/CE-3292

CE-3292: Added translations for some special pages

lukaszkonieczny committed:
CE-2505: Move hook to ProtectSiteJS extension

lukaszkonieczny committed:
CE-2505: Remove accidentally added changes

lukaszkonieczny committed:
CE-2505: Add doc block

Commit:
Merge branch 'dev' into SUS-75

lukaszkonieczny committed:
CE-2505: Add spaces in array brackets

Commit:
Merge pull request #9591 from Wikia/SUS-75

SUS-75 Check type of a file supplied to the Video Embed Tool

lukaszkonieczny committed:
Merge pull request #9406 from Wikia/CE-3020

Script to move js pages in main namespace to new namespace

Commit:
Merge pull request #9582 from Wikia/SUS-57

SUS-57 fix broken link on Special:PhalanxStats by changing the MW message to…

Commit:
Merge pull request #9590 from Wikia/ADEN-2292-openx-provider-removal

ADEN-2292 OpenX provider removal

vforge committed:
XW-1036 | Make sure localStorage API is there

Commit:
RECIRC-39: Don't show recirc module when noexternals=1

Commit:
VE-2122: Removed all the tracking related to issue PLATFORM-1355

inez committed:
Merge pull request #9599 from Wikia/VE-2122

VE-2122: Removed all the tracking related to issue PLATFORM-1355

Commit:
VE-2123: Track if PCRE errors are actually happening

This is a step towards starting throwing exceptions when those errors are happening

Commit:
VE-2124: Make sure to set $wgArticleAsJson back to false even if Exception is thrown

ArticleApiController might be called via FauxRequest which does not separates scopes between caller and callee - so this potentially could affect caller scope.

Commit:
Merge pull request #9598 from Wikia/RECIRC-39

RECIRC-39: Don't show recirculation rail when noexternals=1

kamilkoterba committed:
CE-3262 Adding translations improvements after localization testing phase

vforge committed:
XW-1036 | Add explanation and link to tickets

vforge committed:
XW-1036 | Less conflict-prone name

Commit:
SERVICES-969 add preferences service to populate data in getOptions

Commit:
Merge pull request #9608 from Wikia/XW-1036

XW-1036 | Make sure we test localStorage API

kamilkoterba committed:
CE-3262 Add missing translations for special pages names

Commit:
Merge pull request #9586 from Wikia/SUS-48

SUS-48, remove LVS

adamkarminski committed:
Merge pull request #9610 from Wikia/CE-3262-translations

CE-3262 translations

Commit:
SERVICES-969 add test that check is getOption provide preference from service

kamilkoterba committed:
CE-3262 Adding translations updates for template classification

Commit:
SERVICES-1137 Remove PreferenceCorrectionService

adamkarminski committed:
Merge pull request #9613 from Wikia/CE-3262-translations-2

CE-3262 Adding translations updates for template classification

kamilkoterba committed:
CE-3197 Remove unnecessary method

Commit:
Merge pull request #9612 from Wikia/SERVICES-969

Services 969

Commit:
Merge pull request #9615 from Wikia/SERVICES-1137

SERVICES-1137 Remove PreferenceCorrectionService

pawelchcki committed:
MercuryImageReview cleanup iframe styles + add temporary hack for iframe height. Fix translations and description.

nmonterroso committed:
resolve helios address using consul rather than a hard-coded dns entry to allow us to switch more easily between the unavailable service

Commit:
RECIRC-28: Update recirculation rail taboola style

pawelchcki committed:
fix one typo

Commit:
Add test scaffolding for ExactTargetApi::sendRequest

- add a wrapper class for exposing ExactTargetApi::sendRequest publicly
- add a test to make sure that the SoapFault is caught
- fix the reference to SoapFault to make the test pass

Commit:
Add happy path test case and test logger calls

inez committed:
Merge pull request #9601 from Wikia/VE-2124

VE-2124: Make sure to set $wgArticleAsJson back to false even if Exception is thrown

Commit:
MAIN-6319: move logEmailChanges hook to a global location

Commit:
RECIRC-7: Create a data service for discussions API

Commit:
RECIRC-7: Build discussions template with styles

Commit:
RECIRC-7: Load discussions asynchronously

kamilkoterba committed:
CE-3305 Fix lang code typo

kamilkoterba committed:
CE-3305 Add missing ja translation

kamilkoterba committed:
CE-3305 i18n for flags global shortcuts

Commit:
more informative log message

Commit:
use __METHOD__ as log message label

Commit:
fix whitespace

Commit:
update unit tests

kamilkoterba committed:
CE-3305 Distribute messages properly to extensions

Commit:
Merge pull request #9619 from Wikia/SUS-34-test

WIP Add test scaffolding for ExactTargetApi::sendRequest

pawelchcki committed:
address cr

pawelchcki committed:
Merge pull request #9617 from Wikia/fix_image_review_mercury

PLATFORM-1331 MercuryImageReview cleanup iframe styles

adamkarminski committed:
Merge pull request #9624 from Wikia/CE-3305-translations-fixes

Ce 3305 translations fixes

Grunny committed:
Merge pull request #9620 from Wikia/MAIN-6319

MAIN-6319: move logEmailChanges hook to a global location

macbre committed:
SUS-76: force JSON response format in VideoHandlerController

macbre committed:
VideoPageToolAssetTest: make the integration test more reliable

Actually wait for DB slave to catch up so that VideoPageToolAsset::newAsset can load a row from slave node.

Use wfWaitForSlaves() instead of sleep(1). When I wait for a bus it does not always take exactly 5 minutes ;)

macbre committed:
Merge pull request #9628 from Wikia/SUS-118-VideoPageToolAssetTest-really-wait-for-slaves

VideoPageToolAssetTest: make the integration test more reliable

vforge committed:
XW-980 | Remove GN pre-2016 code

Commit:
ADEN-2917 no proxy for external asset requests

inez committed:
Merge pull request #9600 from Wikia/VE-2123

VE-2123: Track if PCRE errors are actually happening

Commit:
RECIRC-7: First round of CR fixes

Commit:
Merge pull request #9616 from Wikia/RECIRC-28

RECIRC-28: Update recirculation rail taboola style

Commit:
Create a failing test for ExactTargetApiHelper

The wrapRetriveRequest method fails with "Class
'ExactTarget_RetrieveRequest' not found". Add a test so we can fix it.

Commit:
SEO-115 Favicon is missing on section redirects

There seems to be an issue with Firefox handling of favicons in
combination with pdating the location.hash. "Reloading" the
`<link rel="shortcut icon">` seems enough to fix it

Commit:
VE-2126: Log VE<->Parsoid request duration and allow for switching caching off

Switching caching off is needed just for our future experiment to find out if degraded performance is actually affecting our VE related metrics.
Having duration logged will let us verify that indeed performance is degraded.

Commit:
VE-2126: Move variable to CommonSettings.php instead

Commit:
Break out the ExactTarget_* classes into files

There should only be one class per file.

nmonterroso committed:
don't remove a user's access token cookie when $wgAuthForceLogoutOnBadCredentials is false. $wgAuthForceLogoutOnBadCredentials is true by default, false in reston (from config repo)

nmonterroso committed:
renaming wgAuthForceLogoutOnBadCredentials to wgAuthForceLogoutOnFailedAuthSessionFallback

Commit:
RECIRC-7: Switch to using wikifactory based configuration

Commit:
Remove testDeleteSubscriber; it was testing the mock

Remove the testDeleteSubscriber test. It was testing the mock.

Commit:
Fix ExactTargetApiHelperTest name

Commit:
RECIRC-7: Convert spaces to tabs

Commit:
Merge pull request #9621 from Wikia/RECIRC-7

RECIRC-7: AB test discussions at the bottom of article pages

vforge committed:
XW-980 | Move scss rules

Commit:
remove unused parameter from processResponse() methods

azarek committed:
Merge pull request #9593 from Wikia/CE-2505

CE-2505: Do not allow move js pages to MW or USER namespace

Commit:
log exceptions and context together

vforge committed:
XW-980 | Finish styling changes, remove obsolete menu-aim

vforge committed:
XW-980 | Fix tests

vforge committed:
XW-980 | Re-add (temporarily) wgVar

vforge committed:
XW-980 | Fix inverse

Commit:
Merge branch 'dev' into PLATFORM-1836-Disable-Special-PasswordReset

Commit:
Disabling PasswordReset by username

vforge committed:
XW-980 | Global Nav scss fixes

Commit:
Merge pull request #9636 from Wikia/SUS-74-video-foreign-db-file-exception

SUS-74 `UpdateThumbnailTask::retryThumbUpload()` was throwing exceptions

Commit:
Changed way we disable this special page

Commit:
Merge pull request #9629 from Wikia/XW-980

XW-980 | Remove GlobalNavigation pre-2016 code

macbre committed:
Merge branch 'dev' into SUS-76-VideoHandlerController-force-json-format

RafalWilinski committed:
Merge pull request #9637 from Wikia/PLATFORM-1836-Disable-Special-PasswordReset

PLATFORM-1836 Disable Special:PasswordReset

Commit:
Merge pull request #9627 from Wikia/SUS-76-VideoHandlerController-force-json-format

SUS-76: force JSON response format in VideoHandlerController

Commit:
Merge pull request #9631 from Wikia/ADEN-2917-no-proxy-for-source-point-recovery

ADEN-2917 No proxy for external asset requests

Commit:
Add test for prepareSubscriberObjects

nmonterroso committed:
Merge pull request #9634 from Wikia/services-1125

Helios Service Resolution and Cookie Deletion

Commit:
Remove the note about loading ExactTargetSoapClient first

Removed the note about loading ExactTargetSoapClient first. It's no
longer necessary since each class can be loaded individually as
referenced.

Commit:
Add a test for wrapCreateRequest

inez committed:
Merge pull request #9633 from Wikia/VE-2126

VE-2126: Log VE<->Parsoid request duration and allow for switching caching off

Commit:
add a test for wrapSimpleFilterPart

Commit:
SEO-195 Add global variable to include environment in html title

Commit:
SEO-206 Cache new robots.txt for 24 hours

drsnyder committed:
Merge pull request #9635 from Wikia/SUS-129-test

Fix ExactTarget_* class autoloading

Commit:
RECIRC-47: Add in `recent_popular` option for Fandom posts

Commit:
Merge pull request #9632 from Wikia/SEO-115-favicons-section-redirects

SEO-115 Favicon is missing on section redirects

Commit:
SEO-118 VE: Hide or remove "Let this page be indexed by search engines" from Options dialog

Removing the "indexing" section. It was verified having __INDEX__ or
__NOINDEX__ in the source has no result on actual robots directive.

Commit:
Cast duration to int so it is correctly picked up as long by elastic search

Commit:
SEO-195 Just one if clause

inez committed:
Merge pull request #9645 from Wikia/cast-duration-to-int

Cast duration to int so it is correctly picked up as long by elastic search

Commit:
Merge pull request #9639 from Wikia/SEO-195-wg-var-for-controlling-hostname-in-html-title

SEO-195 Add global variable to include hostname in HTML title

Commit:
Merge pull request #9641 from Wikia/SEO-206-new-cache-robots-txt-for-24-hours

SEO-206 Cache new robots.txt for 24 hours

Commit:
Merge pull request #9642 from Wikia/RECIRC-47

RECIRC-47: Add in `recent_popular` option for Fandom posts

Commit:
Merge pull request #9643 from Wikia/SEO-118-remove-indexing-options-from-ve

SEO-118 VE: Hide or remove "Let this page be indexed by search engines" from Options dialog

Commit:
Merge branch 'dev' into SOC-1840
